css/style.css

```diff
@@ -411,13 +719,178 @@
     padding-right: 35px;
   }
 
+  .data__figures {
+    font-weight: 600;
+  }
+
   .test-btn {
     padding-right: 0;
   }
 
   .testdrive {
     margin-right: 60px;
   }
+
+  .info__img {
+    height: auto;
+    width: 100%;
+    max-width: 1199px;
+  }
+
+  .wrap-information, .wrap {
+    height: 747px;
+    }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)